### PR TITLE
HZP-283: Initial version of ada2fhir ELZ

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht.xml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      <display value="Patient, Sandra Janine (Sandra) van XXX_Sanderink"/>
+   </subject>
+   <date value="T"/>
+   <author>
+      <reference value="PractitionerRole/practitionerrole-000001111"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role), Huisartsen, niet nader gespecificeerd, Gezondheidscentrum de Brug"/>
+   </author>
+   <title value="Referral note"/>
+   <section>
```
Comment (NictizEduard) [inline ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht.xml:30]: The .text values are missing in the sections.

File: ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <display value="Patient,"/>
+   </subject>
+   <date value="T"/>
+   <author>
+      <reference value="PractitionerRole/practitionerrole-000001111"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role)"/>
+   </author>
+   <title value="Referral note"/>
+   <section>
```
Comment (NictizEduard) [inline ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml:30]: The .text values are missing in the sections.

File: ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht.xml (hunk 1 above)
Comment (LuudSlagter) [reply ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht.xml:30]: Solved!

File: ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml (hunk 2 above)
Comment (LuudSlagter) [reply ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml:30]: Solved!

**Review by NictizEduard — CHANGES_REQUESTED** [review]

File: ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml
[the remark this answers is not in the document]
Comment (LuudSlagter) [reply ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml:1]: I've added corresponding logic in the latest commit!

**Review by NictizEduard — APPROVED** [review]
I approve this PR. @ogieling could you please have a short look if you agree with ADA instances?

**Review by niekvangalen — APPROVED** [review]
I took a brief look and it looks good, approved.

**Review by niekvangalen — APPROVED** [review]

**Pull request merged**